python/cudf_polars/tests/experimental/test_sink.py

```diff
@@ -0,0 +1,57 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025, NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import pytest
+
+import polars as pl
+
+from cudf_polars.testing.asserts import DEFAULT_SCHEDULER, assert_sink_result_equal
+from cudf_polars.utils.versions import POLARS_VERSION_LT_128
+
+
+@pytest.fixture(scope="module")
+def df():
+    return pl.LazyFrame(
+        {
+            "x": range(30_000),
+            "y": [1, 2, None] * 10_000,
+            "z": ["ẅ", "a", "z", "123", "abcd"] * 6_000,
+        }
+    )
+
+
+@pytest.mark.parametrize("mkdir", [True, False])
+@pytest.mark.parametrize("data_page_size", [None, 256_000])
+@pytest.mark.parametrize("row_group_size", [None, 1_000])
+@pytest.mark.parametrize("max_rows_per_partition", [1_000, 1_000_000])
+def test_sink_parquet(
+    request, df, tmp_path, mkdir, data_page_size, row_group_size, max_rows_per_partition
+):
+    request.applymarker(
```

Comment:
I haven't seen `request.applymarker` before. Is there any advantage to using that over `pytest.xfail(reason)` inside the test? https://docs.pytest.org/en/stable/how-to/skipping.html